src/components/Events.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import type { Event } from "../model/Event";
+import style from "./Events.module.scss";
+import React from "react";
+
+export const Events: React.FC = () => {
+  const [events, setEvents] = useState<Event[]>([]);
+
+  useEffect(() => {
+    const apiUrl = "https://sportujspolu-api.onrender.com/api/v1/events";
+
+    fetch(apiUrl)
+      .then((response) => {
```

Comment:
Ok, you have tried pre-es6 promise-then approach. It looks fine and works. Let's replace it now with modern async-await and let's stick solely to it